assets/app/images/replicator-icon.svg

```diff
@@ -0,0 +1,27 @@
+<?xml version="1.0" encoding="UTF-8" standalone="no"?>
```

Comment:
Why are the SVGs included here instead of being included in kubernetes-ui/topology-graph?